common/src/edgeai_demo.cpp

```diff
@@ -0,0 +1,1147 @@
+                                             &m_cameraObj->ldcObj,
+                                             m_cameraObj->vissObj.output2.arr[0],
+                                             TIVX_TARGET_VPAC_LDC1);
+        }
+    }
+
+    if(m_cameraObj != NULL)
+    {
+        /* Create multiscaler module with appropriate msc connected to ldc node.*/
+        for(auto &iter : m_camMscIdxMap)
+        {
+            if(status == VX_SUCCESS)
+            {
+                status = tiovx_multi_scaler_module_create(m_ovxGraph->graph,
+                                                          &m_multiScalerObjs[iter]->multiScalerObj1,
+                                                          m_cameraObj->ldcObj.output0.arr[0],
+                                                          TIVX_TARGET_VPAC_MSC1);
+
+                if(status == VX_SUCCESS && m_multiScalerObjs[iter]->useSecondaryMsc)
+                {
+                    status = tiovx_multi_scaler_module_create(m_ovxGraph->graph,
+                                                              &m_multiScalerObjs[iter]->multiScalerObj2,
+                                                              m_multiScalerObjs[iter]->multiScalerObj1.output[0].arr[0],
+                                                              TIVX_TARGET_VPAC_MSC2);
+                }
+
+                /* If the input is camera, the multiscaler used with this
+                 * subflow is not the first node.
+                 */
+                m_multiScalerObjs[iter]->isFirstNode = false;
+            }
+        }
+    }
+
+    for(uint i=0; i < m_multiScalerObjs.size(); i++)
+    {
+        /* Create multiscaler module with NULL input in case it is the first node.
+         * readImage will be used to feed the nodes with frames.
+         */
+        if(status == VX_SUCCESS && m_multiScalerObjs[i]->isFirstNode)
+        {
+            status = tiovx_multi_scaler_module_create(m_ovxGraph->graph,
+                                                      &m_multiScalerObjs[i]->multiScalerObj1,
+                                                      NULL,
+                                                      TIVX_TARGET_VPAC_MSC1);
+
+            if(status == VX_SUCCESS && m_multiScalerObjs[i]->useSecondaryMsc)
+            {
+                status = tiovx_multi_scaler_module_create(m_ovxGraph->graph,
+                                                          &m_multiScalerObjs[i]->multiScalerObj2,
+                                                          m_multiScalerObjs[i]->multiScalerObj1.output[0].arr[0],
+                                                          TIVX_TARGET_VPAC_MSC2);
+            }
+        }
+    }
+
+    /* Create pre process module. */
+    for(uint i=0; i < m_multiScalerObjs.size(); i++)
+    {
+        if(status == VX_SUCCESS)
+        {
+            /**
+             * If two back to back msc is needed , in case pre-proce resolution
```

Comment:
typos in this comment 
"pre-proce"
"tha"
"scalerd"

Reply:
Updated the PR